package.json

```diff
@@ -12,6 +12,20 @@
   "dependencies": {
     "@material-ui/core": "^4.4.0",
     "@material-ui/icons": "^4.2.1",
+    "@testing-library/jest-dom": "^4.2.4",
```

Comment:
Should these be in devDependencies?

Reply:
https://github.com/facebook/create-react-app/issues/6180#issuecomment-453640473